float64 t_int
float64 xdes
float64 ydes
float64 phi
float64 theta
float64 phinowrap
float64 t_intorno
float64 t_arrivo
